packages/reaction-email-templates/templates.js

```diff
@@ -1,40 +0,0 @@
-/**
- * ReactionEmailTemplate - Returns a template source for SSR consumption
- * layout must be defined + template
- * @param {String} template name of the template in either Layouts or fs
- * @returns {Object} returns source
- */
-ReactionEmailTemplate = function (template) {
-  check(template, String);
-  let source;
-  let lang = "en";
-
-  const shopLocale = Meteor.call("shop/getLocale");
-
-  if (shopLocale && shopLocale.locale && shopLocale.locale.languages) {
-    lang = shopLocale.locale.languages;
-  }
-
-  // using layout where in the future a more comprehensive rule based
-  // filter of the email templates can be implemented.
-  const tpl = ReactionCore.Collections.Packages.findOne({
-    "layout.template": template
-  });
-
-  if (tpl) {
```

Comment:
This functionality was here to allow for language specific templates, that could be edited in a UI (that is yet to be built).  By removing this we'd be moving to only being able to edit the translations.  This may be ok just for translations, but wouldn't allow for the easy customization of the layout of the templates.